data/cookbook/web-programming/simple-http-client/00-camlnet.ml

```diff
@@ -0,0 +1,26 @@
+---
+packages:
+- name: "camlnet"
+  version: "4.1.9-2"
+- name: "conf-gnutls"
+  version: "1"
+libraries:
+- netclient
+  nettls-gnutls
+discussion: |
+  - **Understanding `ocamlnet`:** The `ocamlnet` provides a rich set of networking libraries that cover many protocols. It provides some modules with a rich set of parameters. But for simple needs, the `Nethttp_client.Convenience` provides simple functions for basic interaction with the server. Note: These functions don't handle redirections.
+  - **Reference:** The `Convenience` module used for simple HTTP interaction is documented on [the ocamlnet site](http://projects.camlcity.org/projects/dl/ocamlnet-4.1.9/doc/html-main/Nethttp_client.Convenience.html)
```

Comment:
Is `ocamlnet` the official term (in monospace). I didn't see it on that linked website. If so, this one also needs to be in monospace. If not, it would follow the OCaml format used in OBuilder, OCamlFormat, etc., to write it as OCamlNet. @cuihtlauac, what do you think?

Reply:
`ocamlnet` is the name of the package from a, Opam view. [The Ocamlnet](http://projects.camlcity.org/projects/ocamlnet.html) spell it Ocamlnet.

Reply:
The Ocamlnet library has been changed for cohttp-lwt... it doesn't fit in a 2 lines program anymore...